docker/test/integration/cluster/checkers/PrometheusChecker.py

```diff
@@ -18,7 +18,16 @@
 
 class PrometheusChecker:
     def __init__(self):
-        self.prometheus_client = PrometheusConnect(url="http://localhost:9090", disable_ssl=True)
+        self.use_ssl = False
+
+    def enable_ssl(self):
+        self.use_ssl = True
+
+    def _getClient(self):
+        if self.use_ssl:
+            return PrometheusConnect(url="https://localhost:9090", disable_ssl=True)
```

Comment:
We don't want to use SSL verification for the python client when connecting from this checker. We only need SSL verification between the MiNiFi and the Prometheus server.

Reply:
Do we even need the option of using SSL on the prometheus listener? It doesn't interact with minifi, since prometheus is the client in the metrics collection, and it doesn't need client certificates either. Why don't we just throw out the ssl option in PrometheusChecker altogether?